index.html

```diff
@@ -762,18 +845,145 @@ <h2><dfn>CaptureController</dfn></h2>
             };
           </pre>
           <dl data-link-for="CaptureController" data-dfn-for="CaptureController" class="methods">
             <dt>
               <dfn>constructor</dfn>
             </dt>
+            <dd>
+              Run the following steps:
+              <ol>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\IsBound]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot tracks whether [=this=] has been associated with a [=capture-session=].
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\Source]]</dfn>,
+                    initialized to <code>null</code>.
+                    <div class="note">
+                      This internal slot tracks with which source the [=capture-session=] is associated.
+                      It is distinct from {{CaptureController/[[IsBound]]}}, in that {{CaptureController/[[IsBound]]}}
+                      is set even if {{MediaDevices/getDisplayMedia()}} rejects, whereas {{CaptureController/[[Source]]}}
+                      will only be set if {{MediaDevices/getDisplayMedia()}} succeeds.
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\DisplaySurfaceType]]</dfn>,
+                    initialized to <code>null</code>.
+                    <div class="note">
+                      Once capture starts, this will be set to the type of the capture [=display surface=].
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\InitialFocusLost]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot ensures that calls to {{CaptureController/setFocusBehavior()}} fail
+                      without throwing if the user manually focuses another tab or native application.
+                    </div>
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Create one internal slot: <dfn data-dfn-for="CaptureController">[[\FocusWindowClosed]]</dfn>,
+                    initialized to <code>false</code>.
+                    <div class="note">
+                      This internal slot tracks explicit closure of the window of opportunity, which happens when
+                      {{CaptureController/setFocusBehavior()}} is called, or the task to close the window executes.
```

Comment:
Can we link "the task to close the window" to the "Schedule a task to..."?

Reply:
Sure, that does make it easier for a reader to find their way around the doc.